forch/device_testing_server.py

```diff
@@ -1,13 +1,14 @@
 """gRPC server to receive device testing result"""
 
 from concurrent import futures
-import grpc
 import logging
 
+import grpc
+
 import forch.proto.grpc.device_testing_pb2_grpc as device_testing_pb2_grpc
 from forch.proto.shared_constants_pb2 import Empty
 
-LOGGER = logging.getLogger('cstate')
+LOGGER = logging.getLogger('grpcserver')
```

Comment:
dtservicer?  This is more specific to device-testing than it is to grpc.  What happens if there's another grpc RPC somewhere?

Reply:
Updated.